README.md

```diff
@@ -166,16 +158,18 @@ monitors := []Monitor{
 }
 ```
 
-###### Client Create into GreptimeDB
+###### Create into GreptimeDB
 
 ```go
 resp, err := cli.Create(context.Background(), monitors)
```

Comment:
`Create` isn't as good as `Write`, it is confused about creating object locally or writing data to server, how about `cli.ORMWrite(...)`?

Reply:
`ORM` is not a good option. Maybe merge `Create` and `Write` into one `Write` API is a better choice, I will try it after this PR is merged.

Reply:
Java's [API](https://github.com/GreptimeTeam/greptimedb-ingester-java/blob/main/ingester-example/src/main/java/io/greptime/StreamWritePOJOsQuickStart.java#L55) is a great example. Maybe we can learn something from it.

Reply:
[GORM](https://gorm.io/docs/) is a better package to refer for naming convention.